aries-site/src/pages/design-tokens/all-design-tokens.js

```diff
@@ -0,0 +1,342 @@
+      />
+      <Collapsible open={open}>
+        <Box pad={{ left: 'medium' }} flex={false}>
+          {Object.keys(tokensObj[collection]).map(category => (
+            <Button
+              key={category}
+              align="start"
+              label={category}
+              active={active === `${collection}.${category}`}
+              onClick={() => setActive(`${collection}.${category}`)}
+            />
+          ))}
+        </Box>
+      </Collapsible>
+    </Box>
+  );
+};
+const Nav = ({ active, setActive, tokens: tokensObj }) => {
+  return Object.keys(tokensObj).map(collection => (
+    <NavSection
+      key={collection}
+      tokens={tokensObj}
+      collection={collection}
+      active={active}
+      setActive={setActive}
+    />
+  ));
+};
+
+const ColorPreview = ({ datum }) => (
+  <Box
+    pad="medium"
+    round="xsmall"
+    flex={false}
+    background={datum.value}
+    border={{ color: 'border-weak' }}
+  />
+);
+
+const DimensionPreview = ({ datum }) =>
+  parseInt(datum.value, 10) <= 200 ? (
+    <Box
+      round="xsmall"
+      flex={false}
+      background="brand"
+      width={datum.value}
+      height={datum.token.includes('icon') ? datum.value : '36px'}
+    />
+  ) : (
+    '--'
+  );
+
+const RadiusPreview = ({ datum }) => (
+  <Box pad="medium" round={datum.value} flex={false} background="brand" />
+);
+
+const BorderPreview = ({ datum }) => (
+  <Box
+    height="72px"
+    width="144px"
+    round="xsmall"
+    border={{ size: datum.value }}
+    flex={false}
+  />
+);
+
+const WeightPreview = ({ datum }) => (
+  <Text size="xlarge" weight={datum.value}>
+    Aa
+  </Text>
+);
+
+const TextPreview = ({ datum }) => <Text size={datum.value}>Hello world</Text>;
+
+const getTokens = (tokenObj, mode) =>
+  Object.keys(tokenObj).map(key => {
+    return {
+      id: key,
+      token: key,
+      type: tokenObj[key]?.modes[mode]?.$type,
+      description: tokenObj[key]?.modes[mode]?.comment,
+      value: tokenObj[key]?.modes[mode].value,
+    };
+  });
+
+const AllTokens = () => {
+  const [active, setActive] = useState('semantic.color');
+  const [data, setData] = useState([]);
+  const [modes, setModes] = useState([]);
+  const [selectedMode, setSelectedMode] = useState('');
+
+  useEffect(() => {
+    const keyPath = active.split('.');
+
+    let res = structuredTokens;
+    keyPath.forEach(key => {
+      res = res[key];
+    });
+
+    const nextModes = Object.keys(Object.values(res)[0].modes).map(
+      mode => mode,
+    );
+    let nextMode;
+    if (nextModes.includes('light')) nextMode = 'light';
+    else if (nextModes.includes('large')) nextMode = 'large';
+    else [nextMode] = nextModes;
+
+    const nextData = getTokens(res, nextMode);
+
+    setData(nextData);
+    setModes(nextModes);
+    setSelectedMode(nextMode);
+  }, [active]);
+
+  return (
+    <Page kind="full">
+      <BetaNotification />
+      <Box direction="row" gap="large">
+        <Box
+          width="medium"
```

Comment:
Allow context to flex

```suggestion
          width={{max: 'medium'}}
```

Reply:
This made things too narrow, but I improved the responsive behavior of this page.